applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -452,9 +453,9 @@ class ExplicitCentralDifferencesScheme
 
         // Solution of the explicit equation:
         if (nodal_mass > numerical_limit)
-            r_current_acceleration = r_current_residual / nodal_mass;
+            noalias(r_current_acceleration) = (r_current_residual - nodal_displacement_damping * r_current_velocity) / nodal_mass;
         else
-            r_current_acceleration = ZeroVector(3);
+            noalias(r_current_acceleration) = ZeroVector(3);
 
         bool fix_displacements[3] = {false, false, false};
```

Comment:
do not use C arrays(use std::array if you need this functionality)

Reply:
In my defense this was already implemented, but you are right